font/bmfont.ksy

```diff
@@ -0,0 +1,155 @@
+meta:
+  id: bmfont
+  title: BMFont binary file
+  file-extension: fnt
```

Comment:
It'd be nice to fill in more info, like the apps using the format.

Reply:
Added a bit